io/net/sriov_device_test.py

```diff
@@ -77,7 +77,8 @@ def setUp(self):
 
         if self.params.get('netmasks'):
             self.prefix = self.netmask_to_cidr(self.netmask[0])
-        self.peer_ip = self.params.get('peer_ip', default="").split(' ')
+        self.peer_ipaddr = self.params.get(
+            'peer_ipaddr', default="").split(' ')
```

Comment:
@PavamanSubramaniyam I think changing only `self.params.get(
            'peer_ipaddr', default="").split(' ')` is fine.
do not change the assigned variable name. just change the value in yaml and the one in the `self.params.get('peer_ip', default="").split(' ')`

let me know if you need any help here